src/bitcoin-cli.cpp

```diff
@@ -1175,6 +1175,12 @@ static int CommandLineRPC(int argc, char *argv[])
                 fputc('\n', stdout);
             }
         }
+        if (gArgs.IsArgSet("-getinfo") +
+            gArgs.GetBoolArg("-netinfo", false) +
```

Comment:
What's the rationale for using `IsArgSet()` for `getinfo` but `GetBoolArg()` for the others?  Maybe I'm overlooking something simple.

Tried the following locally and didn't see any issues with it:
```diff
diff --git a/src/bitcoin-cli.cpp b/src/bitcoin-cli.cpp
index 4002424af0..b682544fa2 100644
--- a/src/bitcoin-cli.cpp
+++ b/src/bitcoin-cli.cpp
@@ -1175,10 +1175,11 @@ static int CommandLineRPC(int argc, char *argv[])
                 fputc('\n', stdout);
             }
         }
+
         if (gArgs.IsArgSet("-getinfo") +
-            gArgs.GetBoolArg("-netinfo", false) +
-            gArgs.GetBoolArg("-generate", false) +
-            gArgs.GetBoolArg("-addrinfo", false) > 1) {
+            gArgs.IsArgSet("-netinfo") +
+            gArgs.IsArgSet("-generate") +
+            gArgs.IsArgSet("-addrinfo") > 1) {
             throw std::runtime_error("Only one of \"-getinfo\", \"-netinfo\", \"-generate\", and \"-addrinfo\" may be specified.");
         }
         std::unique_ptr<BaseRequestHandler> rh;
```

Reply:
I wasn't keen on choosing between `IsArgSet() `and `GetBoolArg() `since both achieve the same desired outcome, but you are certainly right; using either of the two, for all would be a better approach.